gustaf/utils/connec.py

```diff
@@ -274,8 +280,15 @@ def make_quad_faces(resolutions):
     nnpd = np.asarray(resolutions)  # number of nodes per dimension
     total_nodes = np.product(nnpd)
     total_faces = (nnpd[0] - 1) * (nnpd[1] - 1)
-    node_indices = np.arange(total_nodes).reshape(nnpd[1], nnpd[0])
-    faces = np.ones((total_faces, 4)) * -1
+    try:
+        node_indices = np.arange(total_nodes).reshape(nnpd[1], nnpd[0])
+    except ValueError as e:
+        raise ValueError(f"Problem with generating node indices. {e}")
+
+    try:
+        faces = np.ones((total_faces, 4)) * -1
+    except ValueError as e:
+        raise ValueError(f"Problem with generating faces. {e}")
```

Comment:
when do you expect this to fail? It seems that as long as numpy is correctly installed, this will work

Reply:
This exception is thrown by the reshape function. E.g. the total_nodes is 60 and nnpd = (4, 3) the exception is thrown.

Reply:
yes, I get the first try-except. but the question was for the second one with `np.ones()`